vignettes/vignette_best_practices.Rmd

```diff
@@ -0,0 +1,198 @@
+---
+title: "Best practices when writing vignette in admiral"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Best practices when writing vignette in admiral}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup0, include = FALSE}
```

Comment:
We have `r_setup0` here but `r_setup` below in the default options.  Why?

Reply:
updated thanks